src/AutoBuild.jl

```diff
@@ -920,6 +930,7 @@ function rebuild_jll_package(obj::Dict;
         upload_prefix;
         verbose=verbose,
         lazy_artifacts = lazy_artifacts,
+        init_block = get(obj, "init_block", ""),
```

Comment:
Is this the correct way to do it?

Reply:
We can assume that the only `obj` objects coming in are ones generated by this same BB version; so we will always have an `init_block` key in `obj`.